src/components/Asset/AssetLogo.tsx

```diff
@@ -27,10 +27,8 @@ export const AssetLogo: FC<AssetLogoProps> = ({ className, tokenAddress }) => {
   const [isError, setIsError] = useState(false)
 
   useEffect(() => {
-    if (!isError) {
-      setIsError(tokenAddress === "0x" || tokenAddress === undefined)
-    }
-  }, [isError, tokenAddress])
+    setIsError(tokenAddress === "0x" || tokenAddress === undefined)
```

Comment:
This doesn't need to be handled with state. Let's leave `isError` for handling actual errors with the image loading:

```tsx
const isBadProps = tokenAddress === "0x" || tokenAddress === undefined
// ...
{isError || isBadProps ? <>error code</> : <>success code</>} 
```